src/Compiler/Scheme/Racket.idr

```diff
@@ -42,6 +42,7 @@ schHeader : String -> String
 schHeader libs
   = "#lang racket/base\n" ++
     "; @generated\n" ++
+    "(require racket/async-channel)\n" ++ -- for asynchronous channels
```

Comment:
just a heads up, adding requires will need to wait until next bootstrap release to work. :C

(That is why I didn't use future in the racket support code right away)